### PR TITLE
Re-enable stdout logging for e2e

### PR DESCRIPTION
**Pull request by sebastienvas**
Since the moved the istio logging, the logging to stdout was gone. Removing the glog flags call as they have no effect, and updating the tests/istio.mk to call tests/e2e.sh instead of duplicating its content.

### CODE REVIEW

**Comment by costinm** [comment]
I would rather remove the script ( we are trying to remove scripts that can
be moved to make - there is a bit more consistency)

**Comment by ldemailly** [comment]
fixes #2483 ?

File: tests/istio.mk [hunk 1]
```diff
@@ -42,35 +42,28 @@ e2e_docker: docker push
 endif
 
 E2E_ARGS ?=
-E2E_ARGS += $(if ifeq($V,1),-alsologtostderr -test.v -v 2)
 E2E_ARGS += ${MINIKUBE_FLAGS}
 E2E_ARGS += --istioctl ${GOPATH}/bin/istioctl
 
+EXTRA_E2E_ARGS = --mixer_tag ${TAG}
```
Comment (costinm) [inline tests/istio.mk]: I think someone added a flag that applies for all at once ?

**Comment by ldemailly** [comment]
Sorry I'm a bit confused,
Are we capturing only stdout ? only stderr ? both ?
what controls it ? can it be both ?
(somehow I don't see the fortio output that used to show up)

**Comment by sebastienvas** [comment]
This just changes the logging to log to stdout as well as in a file.

**Comment by ldemailly** [comment]
I was mistaken because the t.log and log go to different part of the output
cleared up now that all 3 of stdout, stderr and log output do go to the prow logs correctly (thanks seb!)

File: prow/e2e-suite.sh [hunk 2]
```diff
@@ -69,13 +70,4 @@ mkdir -p ${GOPATH}/src/istio.io/istio/_artifacts
...
-  --mixer_tag "${GIT_SHA}"\
-  --mixer_hub "${HUB}"\
-  --pilot_tag "${GIT_SHA}"\
-  --pilot_hub "${HUB}"\
-  --ca_tag "${GIT_SHA}"\
-  --ca_hub "${HUB}"\
-  --istioctl ${GOPATH}/bin/istioctl
-
-
+time make e2e_all
```
Comment (mattdelco) [inline prow/e2e-suite.sh]: I could easily be misreading things, but various scripts seem to call this script with parameters like "--auth_enable" and "--cluster_wide".  Some of these were getting passed around via the $@.  What used to be a call to "./tests/e2e.sh $@" has now been replaced with a simple "make e2e_all".  It looks to me like these parameters are now being discarded/ignored.  I'm thinking what might be needed is something like this:
make e2e_all TESTOPTS="$@"
or maybe just explicit handling for --auth_enable if --cluster_wide is just intended as a parameter for cluster_lib.sh.

**Pull request merged**